app/controllers/content/index.js

```diff
@@ -215,41 +215,11 @@ export default Controller.extend(Analytics, {
             window.open(href, '', 'menubar=no,toolbar=no,resizable=yes,scrollbars=yes,width=600,height=400');
             return false;
         },
-        // Sends Event to GA/Keen as normal. Sends second event to Keen under
-        // "non-contributor-preprint-downloads" collection to track non contributor
-        // preprint downloads specifically.
-        dualTrackNonContributors(category, label, url, primary) {
+        // Sends Event to GA.  Previously sent a second event to Keen to track non-contributor
+        // downloads, but that functionality has been removed.  Stub left in place in case we want
+        // to double-log later.
+        dualTrackNonContributors(category, label, url) {
```

Comment:
Maybe rename this if it's no longer dual tracking.

Reply:
Good call!